packages/marshal/src/encodePassable.js

```diff
@@ -728,6 +728,7 @@ const makeInnerDecode = (decodeStringSuffix, decodeArray, options) => {
       }
     }
   };
+  // @ts-expect-error XXX Type 'unknown' is not assignable to type 'Passable<PassableCap, Error>'.
```

Comment:
```suggestion
  // @ts-expect-error Type 'unknown' is not assignable to type 'Passable<PassableCap, Error>'.
```
I think this is fine without elaboration.